### ISSUE TITLE
Drawer-layout doesn't set contentContainer margin when element is loaded dynamically

### ISSUE THREAD

**Issue by ronnyroeller** [PolymerElements/app-layout#387]
### Description

We load the app element dynamically via `Polymer.Base.importHref` to improve time of first render. On desktops (where the drawer is opened by default) this leads to the drawer overlaying the content area.

The issue occurs in Chrome (54.0.2840.98) and Canary (57.0.29380.0)..

### Expected outcome

Content area is moved right of the drawer.

![image](https://cloud.githubusercontent.com/assets/9339055/20797308/7d4838f0-b7da-11e6-8781-a1159f706cb1.png)

### Actual outcome

Drawer overlays the content area (see how the text is cut). Triggering a resize (e.g. by changing browser width or via `notifyResize` renders it correctly.

![image](https://cloud.githubusercontent.com/assets/9339055/20797312/837ec7de-b7da-11e6-9ed9-ef32863a05e5.png)

### Live Demo
[JSbin currently returns a "Cross-Origin Resource Sharing" error when importing Polymer elements from polygit.org. So I'll post here the code itself.]

The app element with the `<app-drawer-layout>`
```html
<!doctype html>
<base href="bower_components/">
<link rel="import" href="polymer/polymer.html">
<link rel="import" href="app-layout/app-drawer-layout/app-drawer-layout.html">
<link rel="import" href="app-layout/app-drawer/app-drawer.html">

<dom-module id="my-app">
	<template>
		<app-drawer-layout >
			<app-drawer>
				<div>My drawer</div>
			</app-drawer>

			<div>Lots of content. Lots of content. Lots of content. Lots of content. Lots of content. Lots of content. Lots of content.</div>
		</app-drawer-layout>
	</template>
</dom-module>

<script>
(function() {
	Polymer({
		is: 'my-app'
	});
})();
</script>
```

The shell page which dynamically loads the app element:
```html
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<base href="bower_components/">
	<script src="webcomponentsjs/webcomponents-lite.js"></script>
	<link rel="import" href="polymer/polymer.html">
</head>
<body>
	<template is="dom-bind">
		<my-app></my-app>
	</template>

	<script>
		setTimeout(function() {
			Polymer.Base.importHref('../my-app.html');
		}, 1);
	 </script>
</body>
</html>
```

### Steps to reproduce

1. Create an app element with a `<app-drawer-layout>`
2. Create a shell page that dynamically loads the app element
3. Open application in a Chrome desktop (so the drawer is opened by default) => Issue is visible
4. Resize browser (to trigger resize event) -> this will show the correct layout

### Browsers Affected
- [x] Chrome Mac
- [x] Chrome Linux
- [x] Canary
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by blasten** [comment]
This is likely a race condition when upgrading `app-drawer-layout` and `app-drawer`. We fixed that problem in 2.0: https://github.com/PolymerElements/app-layout/blob/2.0-preview/app-layout-behavior/app-layout-behavior.html

**Comment by keanulee** [comment]
@blasten @frankiefu Should we backport #340 to the Polymer 1.0 (`master`) branch then?

**Comment by blasten** [comment]
probably.

**Comment by tpluscode** [comment]
I'm having similar issues with drawers without dynamically loading content too. See [this JSBin](https://jsbin.com/fijamocuco/edit?html,output). Initially the drawer is positioned over the main content because the latter's `#contentContainer` has no margin until layout recalculates. I've seen this both with Polymer 1.0 and 2.0.

**Comment by keanulee** [comment]
@tpluscode Can you try that

**Comment by keanulee** [comment]
@tpluscode Can you try that same JSBin in Chrome Canary? It seems to be working there, so something in Chrome Stable may be broke app-drawer-layout 1.x. Also, consider trying `app-layout#2.0-preview` with Polymer 1.0 where this issue should be fixed.

**Comment by keanulee** [comment]
Fixed by #471. With this change, if you specify a custom value for `--app-drawer-width`, that value must be accessible by both elements. This can be done by defining the value on the `:host` that contains <app-drawer-layout> (or `html` if outside a shadow root):

```css
:host {
  --app-drawer-width: 300px;
}
```
